src/components/widgets/ConceptCard.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="concept-item">
+  <div class="concept-item" @click="$emit('click')">
```

Comment:
This `$emit('click')` seems useless, as it was never used in the application.

Reply:
Ok thank you, a commit was missing.